api/api.go

```diff
@@ -819,6 +829,52 @@ func OrbSource(ctx context.Context, log *logger.Logger, cl *client.Client, orbRe
 	return response.OrbVersion.Source, nil
 }
 
+// OrbInfo gets the meta-data of an orb
+func OrbInfo(ctx context.Context, log *logger.Logger, cl *client.Client, orbRef string) (*OrbVersionResponse, error) {
```

Comment:
How come there's both `cl` and a `client` variables in this function?

Reply:
`cl` here refers to the `client.Client` pointer which is passed in as an argument to this function.

`client` refers to the "client" package found here:
https://github.com/CircleCI-Public/circleci-cli/blob/master/client/client.go

Happy for naming suggestions.